### DISCUSSION TITLE
extending git with kart?

### DISCUSSION THREAD

**Discussion by quinnhornblow** [koordinates/kart#500] [Ideas]
Hey,

I often work in traditional git repositories with and mixture of text/code files and geospatial files. It would be great if I had Kart level version control on my geospatial files. I have had a play around with initiating repositories in different ways using kart init --bare but have found that a kart repo wont track non-geospatial files. And I can't initiate a git repo and kart repo with the same remote location because I get the 'Invalid value for directory .. isn't empty' error.

Do you have a recommendation on how to manage repositories with multiple file types using git and kart?

Thanks

**Comment by craigds** [comment]
Thanks for asking! We're definitely intending to add support for storing regular files alongside data, eventually. But no one's working on it yet.


> I can't initiate a git repo and kart repo with the same remote location because I get the 'Invalid value for directory .. isn't empty' error.

Could you describe this in more detail, including what command you're running and the full output it produces? Perhaps we can make this work better for you.
